src/oas.md

```diff
@@ -3372,8 +3372,9 @@ Note that `discriminator` MUST NOT change the validation outcome of the schema.
 
 | Field Name | Type | Description |
 | ---- | :----: | ---- |
-| <a name="property-name"></a>propertyName | `string` | **REQUIRED**. The name of the property in the payload that will hold the discriminating value. This property SHOULD be required in the payload schema, as the behavior when the property is absent is undefined. |
+| <a name="property-name"></a>propertyName | `string` | **REQUIRED**. The name of the discriminating property in the payload that will hold the discriminating value. The discriminating property may be defined as required or optional, but when defined as optional the Discriminator Object must include a `defaultMapping` field that specifies which schema is expected to validate the structure of the model when no discriminating property is present. |
 | <a name="discriminator-mapping"></a> mapping | Map[`string`, `string`] | An object to hold mappings between payload values and schema names or URI references. |
+| <a name="default"></a> defaultMapping | `string` | The schema name or URI reference to a schema that is expected to validate the structure of the model when the discriminating property is not present in the payload. |
```

Comment:
The name `defaultMapping` suggests to me that this is the default whenever there isn't a defined mapping.  So not just when the property is missing, but when the property has a value that cannot otherwise be resolved.  Is this the intention?  If not, I'd suggest a name tying to absence rather than "default."

Reply:
That was not the intention. But it is an interesting idea. This could be accomplished with a slightly different schema for the defaultMapping. Instead of `not: required: ['petType']` it could be:

```
    properties:
      petType:
        not:
          enum: ['dog', 'cat']
```

This might get unwieldy for a schema with many variants, but there is some attractiveness to this.

Thoughts?